machine-learning/ml_tips/titanic_pipeline.yml

```diff
@@ -0,0 +1,60 @@
+# This is the config file for molehill https://github.com/chezou/molehill
```

Comment:
`titanic_pipeline.yml` still exists in repos and no meaning for customers.

Reply:
removed 491e070